libraries/AP_Compass/AP_Compass.cpp

```diff
@@ -1541,6 +1541,18 @@ Compass::get_healthy_mask() const
     return healthy_mask;
 }
 
+bool
+Compass::healthy(void) const
```

Comment:
Is this notation of return type being placed one line above function name intentional?

Reply:
Yes because that is the convention of this file